tests/bulk/bulkwrite-insert-002.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\BulkWrite::insert() $document is MongoDB\BSON\Document
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+
+$document = MongoDB\BSON\Document::fromJSON('{ "_id": 2 }');
+
+$bulk = new MongoDB\Driver\BulkWrite();
+$insertedId = $bulk->insert($document);
```

Comment:
While implementing this test, I realized there's a separate bug where `_id` fields are not generated for Document instances. I opened [PHPC-2216](https://jira.mongodb.org/browse/PHPC-2216) to track that.